### PR TITLE
MIP10c3-SP21

### PR DESCRIPTION
**Pull request by madumas**

### CODE REVIEW

File: libexec/setzer/setzer-price-avaxusd [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  binance
+  okex
+  huobi
+  gateio
+  bitfinex
+)
+
+case $1 in
+  bitfinex) {
+    setzer x-price "$1" avax%3A:usd
```
Comment (NiklasKunkel) [inline libexec/setzer/setzer-price-avaxusd:14]: why is it `avax%3A:usd` ?

File: libexec/setzer/setzer-price-xtzusd [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -e
+pair=xtzusd
+sources=(
+  binance
+  coinbase
+  kraken
+  okex
+  bitfinex
+  bitfinex
```
Comment (NiklasKunkel) [inline libexec/setzer/setzer-price-xtzusd:10]: bitfinex listed twice, hitbtc is missing

**Review by NiklasKunkel — COMMENTED** [review]
Typically for the `sources` we list them in alphabetical order, could you please make sure these follow that pattern.

File: libexec/setzer/setzer-price-avaxusd (hunk 1 above)
Comment (madumas) [reply libexec/setzer/setzer-price-avaxusd:14]: Workaround for the ":" parsing for a 4 letter symbol. Bitfinex requires a ":" in the URL, so I encode it.
`https://api.bitfinex.com/v2/ticker/tAVAX:USD`

File: libexec/setzer/setzer-price-xtzusd (hunk 2 above)
Comment (madumas) [reply libexec/setzer/setzer-price-xtzusd]: fixed

**Pull request merged**